network/geoblock.sh

```diff
@@ -160,22 +172,31 @@ then
     mapfile -t choice <<< "$choice"
 fi
 
+# Enable config
+{
+echo "MaxMindDBEnable On"
+echo "MaxMindDBFile COUNTRY_DB /usr/local/share/maxminddb/GeoLite2-Country.mmdb"
+echo "MaxMindDBFile CITY_DB /usr/local/share/maxminddb/GeoLite2-City.mmdb"
+} > /etc/apache2/mods-available/maxminddb.conf
+
+
 GEOIP_CONF="#Geoip-block-start - Please don't remove or change this line
-<IfModule mod_geoip.c>
-  GeoIPEnable On
-  GeoIPDBFile /usr/share/GeoIP/GeoIPv4.dat
-  GeoIPDBFile /usr/share/GeoIP/GeoIPv6.dat
+<IfModule mod_maxminddb.c>
```

Comment:
Are you sure this mod is still going to run on ubuntu 22.04? If not, we are going to have a regression...

Reply:
Works on Ubuntu 22.04, no errors anyway. Will continue to test. It' not done just yet. :)